platformio.ini

```diff
@@ -71,32 +69,4 @@ extra_scripts =
 	pre:scripts/build/prebuild_www.py
 	pre:scripts/build/prebuild_cppflags.py
 	pre:scripts/build/prebuild_lua.py ; Needed to generate the .cxx file(s)
-build_type = debug
-
-; GPS edition by hardware revisions
-[env:GPS_EDITION_V3_1]
-build_flags =
-	-D OSW_TARGET_PLATFORM_HEADER='"platform/GPS_EDITION_V3_1.h"'
-	-D GPS_EDITION
-	-D PROGMEM_TILES
-	-D BOARD_HAS_PSRAM
-	-mfix-esp32-psram-cache-issue
-	-D OSW_FEATURE_STATS_STEPS
-	-D OSW_FEATURE_WIFI
-	-D OSW_FEATURE_WIFI_APST
-	-D OSW_FEATURE_WIFI_ONBOOT
-build_type = debug
-
-; GPS edition other stuff
-[env:GPS_EDITION_DEV_ROTATED]
-build_flags =
-	-D OSW_TARGET_PLATFORM_HEADER='"platform/GPS_EDITION_V3_1.h"'
-	-D PROGMEM_TILES
-	-D GPS_EDITION_ROTATED
-	-D BOARD_HAS_PSRAM
-	-mfix-esp32-psram-cache-issue
-	-D OSW_FEATURE_STATS_STEPS
-	-D OSW_FEATURE_WIFI
-	-D OSW_FEATURE_WIFI_APST
-	-D OSW_FEATURE_WIFI_ONBOOT
-build_type = debug
```

Comment:
I would prefer to leave them in, but either comment them or remove them from the pipeline. I would _strongly_ recommend you to port the `setMatrix.sh` script to Python, so I (or anybody else) could easily understand it or - as needed here - patch in a "set of ignored" models, which are not built by the CI anymore (like the GPS* stuff).